.env.template

```diff
@@ -0,0 +1,11 @@
+SECRET_KEY=<your-secret-key>
+EMAIL_USER=<email-address>
+EMAIL_PASSWORD=<your-password>
+
+SENDGRID_API_KEY=<your-sendgrid-api-key>
+DB_BACKEND=postgresql
+DB_NAME=<db-name>
+DB_USERNAME=<db-user-username>
+DB_PASSWORD=<db-user-password>
+DB_HOST=localhost
+DB_PORT=5432
```

Comment:
It will be nice if you can use this format:
```
DB_NAME=<your-db-name or osp>
```
Reason: So that new user will know about the default value that we use from templates

Reply:
Done

Reply:
I don't see the changes used here in the files, which Sankalp suggested
> `DB_NAME=<your-db-name or osp>`

Reply:
@Aaishpra I have not pushed the changes from local that's why it's not visible in the file, I'm waiting for the @codesankalp answer for the above question ☝.